foxpuppet/windows/base.py

```diff
@@ -20,16 +27,17 @@ def document_element(self):
         """ Returns the inner DOM window element.
 
         :returns: DOM window element.
+        :return type: A selenium locator
```

Comment:
This doesn't return the locator, it returns a WebElement: https://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webelement.html#module-selenium.webdriver.remote.webelement